airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -336,6 +343,13 @@ class BigQueryValueCheckOperator(_BigQueryDbHookMixin, SQLValueCheckOperator):
     :param sql: SQL to execute.
     :param use_legacy_sql: Whether to use legacy SQL (true)
         or standard SQL (false).
+    :param encryption_configuration: [Optional] Custom encryption configuration (e.g., Cloud KMS keys).
```

Comment:
```suggestion
    :param encryption_configuration: (Optional) Custom encryption configuration (e.g., Cloud KMS keys).
```

nitpick: let's unify the docstring convention in this module.

Reply:
It seems we've not yet unified this in this module. I'll approve this PR, but I would like to wait for this nitpick to be addressed. Thanks!

Reply:
Didn't notice other places 🥲 
Thank you!
I have changed it in every place in this file

Reply:
No worries 🙂  Thank you so much for the prompt update. I'll merge it once CI pass 🙌